stactask/task.py

```diff
@@ -157,6 +157,8 @@ def items(self) -> ItemCollection:
 
     @classmethod
     def validate(cls, payload: Dict[str, Any]) -> bool:
```

Comment:
Yeah I was trying to make it extra backwards compatible since this is a user-defined function. So if it returns False (like it did before) then you get a generic error